src/main/resources/data.sql

```diff
@@ -1,38 +1,38 @@
 --begin state table initialization
-INSERT IGNORE INTO state (id, name) VALUES ('1', 'Paraná');
+INSERT IGNORE INTO state_tb (id, name) VALUES ('1', 'Paraná');
```

Comment:
recomendo ver disso:
https://stackoverflow.com/questions/2401489/execute-method-on-startup-in-spring
para que dai logo q o servidor comece a executar ele já rode todos os scripts de população de dados e etc sem precisar se preocuparem se o nome das Fields estão corretos ;)